pysilcam/tracking/silcamtracker.py

```diff
@@ -0,0 +1,418 @@
+import os
+import numpy as np
+from skimage.feature import match_template
+from skimage.morphology import label, binary_dilation
+from skimage.measure import regionprops
+import pandas as pd
+from tqdm import tqdm
+import pysilcam.process as scpr
+from pysilcam.background import backgrounder
+from pysilcam.fakepymba import silcam_load
+import h5py
+import names
+
+
+class Tracker:
+    '''
+    Class for tracking
+    '''
+
+    def __init__(self):
+        # define some defaults
+        self.av_window = 15
+        self.THRESHOLD = 0.98
+        self.MIN_LENGTH = 500  # microns
+        self.MIN_SPEED = 0.01  # cm/s
+        self.cross_correlation_threshold = 0.2
+        self.PIX_SIZE = 27.532679738562095
+        self.ecd_tolerance = 0
+        self.path = ''
+        self.DATAFILE = ''
+        self.files = None
+        self.track_length_limit = 15
+
+    def initialise(self):
+        print('* INITIALISE')

```

Comment:
Should we be using logger functionality instead of prints? This could also be a follow-up case.

Reply:
yes - please make a follow-up case

Reply:
Issue #91 logged